examples/getting-started/wp-templates/category.tsx

```diff
@@ -1,42 +1,90 @@
 import React from 'react';
 import { useGeneralSettings, usePosts } from '@wpengine/headless/react';
+import { Pagination } from '@wpengine/headless/next';
 import { getApolloClient, getPosts } from '@wpengine/headless';
 import { useRouter } from 'next/router';
 import { GetStaticPropsContext } from 'next';
 import { Footer, Header, Posts } from '../components';
 
-export default function Category(): JSX.Element {
-  const router = useRouter();
-  const urlParts = router.asPath.split('/').filter(Boolean);
-  const category = urlParts[1];
-  const posts = usePosts({
+const POSTS_PER_PAGE = 10;
```

Comment:
This could come from WP's reading settings via a custom query or a future `useReadingSettings()` hook, but I hard-coded it here for now to keep it more simple.